iron-overlay-behavior.html

```diff
@@ -118,6 +119,14 @@
         type: Object
       },
 
+      /**
+       * The HTMLElement that will be firing relevant KeyboardEvents.
+       */
+      keyEventTarget: {
```

Comment:
I kind of liked the fact to set it here because if i set it on `ready` i would trigger observers. I'll update the doc mentioning `overridden from IronA11yKeysBehavior` ;)